modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java

```diff
@@ -267,10 +267,6 @@ public KBArticle addKBArticle(
 
 		_addKBArticleAttachments(userId, kbArticle, selectedFileNames);
 
-		// Workflow
-
-		_startWorkflowInstance(userId, kbArticle, serviceContext);
-
```

Comment:
Can be this the wrong line to be removed after adding this call to method `updateKBArticle`?

Reply:
<img src="https://media4.giphy.com/media/AloJdViOzqEdq/giphy.gif"/>

i deleted too much!!